src/posts/datatree/index.md

```diff
@@ -0,0 +1,104 @@
+
+## What is a DataTree?
+
+Our solution is the new high-level container class `xarray.DataTree`.
+It acts like a tree of linked `xarray.Dataset` objects, with alignment enforced between variables in sibling nodes but not between parents and children. It can be written to and opened from formats containing multiple groups, such as netCDF4 files and Zarr stores.
+
+For more details please see the [high-level description](https://docs.xarray.dev/en/stable/user-guide/data-structures.html#datatree) and the [dedicated page on hierarchical data](https://docs.xarray.dev/en/stable/user-guide/hierarchical-data.html), and the [section on IO with groups](https://docs.xarray.dev/en/stable/user-guide/io.html#groups) in the xarray documentation.
+
+## Deprecation
+
+If you previously had used the `DataTree` prototype in the [`xarray-contrib/datatree` repository](https://github.com/xarray-contrib/datatree), that has now been archived and will no longer be supported. Instead we encourage you to migrate to the implementation of `DataTree` that you can import from xarray, following the [migration guide](https://github.com/pydata/xarray/blob/main/DATATREE_MIGRATION_GUIDE.md).
+
+## Big moves
+
+This was a big feature addition! For a [decade](https://github.com/pydata/xarray/discussions/8462) there have been 3 core public xarray data structures, now there are 4: [`Variable`](https://docs.xarray.dev/en/stable/generated/xarray.Variable.html#xarray.Variable), [`DataArray`](https://docs.xarray.dev/en/stable/generated/xarray.DataArray.html#xarray.DataArray), [`Dataset`](https://docs.xarray.dev/en/stable/generated/xarray.Dataset.html#xarray.Dataset), and now [`DataTree`](https://docs.xarray.dev/en/stable/generated/xarray.DataTree.html#xarray.DataTree).
+
+Datatree represents arguably one of the single largest new features added to xarray in 10 years - the migration of the existing prototype alone added >10k lines of code across [80 PRs](https://github.com/pydata/xarray/pulls?q=is%3Apr+label%3Atopic-DataTree+is%3Aclosed), and the resulting datatree code now contains contributions from at least 25 people.
+
+We also had to resolve some really [gnarly design questions](https://github.com/pydata/xarray/pull/9063) to make it work in a way we were happy with.
+
+## How did this happen?
+
+DataTree didn't get implemented overnight - it was a multi-year effort that took place in a number of steps.
+
+Initially, the xarray team applied for funding from the [Chan-Zuckerberg initiative]() (LINK?) in March 2021 to develop something like datatree, citing bioscience use cases (e.g. [microscopy image pyramids](https://spatialdata.scverse.org/en/latest/design_doc.html)). Unfortunately whilst we've been lucky to [receive CZI funding before](https://chanzuckerberg.com/eoss/proposals/xarray-multidimensional-labeled-arrays-and-datasets-in-python/), on this occasion we didn't win money to work on the datatree idea.
```

Comment:
@jhamman @shoyer @dcherian is there a public link to the proposal we submitted? I had a look on the CZI website but couldn't find one. I think it would be interesting to see.

Reply:
Here you go -> https://doi.org/10.5281/zenodo.5484176

Reply:
Thank you!